app/build.gradle

```diff
@@ -34,6 +39,8 @@ dependencies {
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     testImplementation 'junit:junit:4.+'
+    testImplementation('org.mockito:mockito-core:3.3.3')
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    //api 'com.google.android.material:material:1.4.0-c01'
```

Comment:
Please remove commented code.

Reply:
Code was removed as requested, see image attached on the previous comment for reference.